xknx/remote_value/remote_value.py

```diff
@@ -27,10 +28,14 @@ def __init__(
         device_name=None,
         feature_name=None,
         after_update_cb=None,
+        listening_group_addresses: List[str] = None,
```

Comment:
in your config PR they are called passive_group_addresses if I recall correctly - which I personally prefer. it implies no action. "listening" is more confusable with "state" imho.